composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/monolog-bundle"               : "~2.7",
         "symfony/swiftmailer-bundle"           : "~2.3",
         "symfony/symfony"                      : "~2.7",
-        "twig/extensions"                      : "~1.2"
+        "twig/extensions"                      : "^1.2"
```

Comment:
Because I executed `composer require twig/extensions` without looking first at `composer.json`. We already had that dependency, and Composer changed when executing that `require` command. Should I revert this change?